views/analysis.py

```diff
@@ -544,6 +547,23 @@ def validate_analysis_extent(request):
         return HttpResponseServerError()
 
 
+def calculate_area(request):
+    if request.method != 'POST':
+        return HttpResponseBadRequest()
+
+    if 'area' in request.session:
+        area = request.session['area']
```

Comment:
are sure about this? 
i've tested running analysis twice using chrome on my local without login first.
i also tried using safari.
all the test covered different area within the same hazard-exposure layers.
none of them returned the same value.
so, i think using session will have no impact on anonymous users.

i've utilised `validate_analysis_extent` on this commit.
as you can see, `area` is added to this function.
the main problem when fully using this function is: all the `hazard_id`, `exposure_id`, and `analysis_extent` are lost in `waiting confirmation`.
this is the reason why i came up with session.

when investigating this, i guess it's because the `run_analysis` on `main.html` redirect the page to a new page. 
so, all the values are gone.